ruff.toml

```diff
@@ -0,0 +1,14 @@
+# https://docs.astral.sh/ruff/configuration/#config-file-discovery
+
+# Assume Python 3.9
+target-version = "py39"
```

Comment:
is this the minimal supported version? This isn't what ships with Sage 10.3, right?

Reply:
Thank you for the context :)